utils/durable/orchestrators/index.ts

```diff
@@ -77,6 +75,13 @@ export const createOrchestrator = <I, TNext = TNextDefault>(
     }
   };
 
+export type CallableActivity<
+  I extends unknown = unknown,
+  S extends ActivityResultSuccess = ActivityResultSuccess,
+  // Failures aren't mapped as they are thrown
+  __ extends ActivityResultFailure = ActivityResultFailure
+> = (context: IOrchestrationFunctionContext, input: I) => Generator<Task, S>;
```

Comment:
@balanza are you sure about this statement? I thought activities could return ActivityResultFailure in case they don't want to ask for a retry, or throw and Error in case they want to retry.

Reply:
A `CallableActivity` either return success or throw an `ActivityFailure` https://github.com/pagopa/io-functions-pushnotifications/blob/7921ca44dca5c2d9c4f1a6b477bbc4536b6ca85a/utils/durable/orchestrators/index.ts#L134

Reply:
ok, we are talking about the callableActivity (the orchestrator wrapper) not the activity itself, which can ends with an ActivityResultFailure, ok